spec/unit/mutant/ast/regexp/expand_regexp_ast_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+RSpec.describe Mutant::AST::Regexp, '.expand_regexp_ast' do
+  it 'returns the expanded AST' do
+    # /foo/
```

Comment:
Non blocking: Instead of this comment: what about running the regexp parser / AST transform to get the `parser_ast` ? This way this test is more meaningful and would have canary properties for future changes?

Reply:
I think that makes sense--I have updated this to match. When I was originally writing the spec it felt like it might be clearer to have the raw AST so I started with that but now that I have just swapped it out for `Unparser.parse` I think it's a lot simpler to read and, as you say, it has a slight "canary" property.